src/integrationTest/java/ch/admin/bar/siard2/cmd/usecases/keys/MsSqlUploadDownloadSiardProjectIT.java

```diff
@@ -52,7 +60,9 @@ public void uploadAndDownload_expectNoExceptions() throws IOException, SQLExcept
                 .expectedArchive(expectedArchive)
                 .actualArchive(actualArchive)
                 .assertionModifier(SiardArchiveAssertions.IGNORE_DBNAME) // FIXME ?
-                .assertionModifier(SiardArchiveAssertions.IGNORE_PRIMARY_KEY_NAME) // Probably a DB-restriction (primary key names are generated)
+                .assertionModifier(SiardArchiveAssertions.IGNORE_PRIMARY_KEY_NAME) // DB restriction ?
+                .assertionModifier(SiardArchiveAssertions.IGNORE_FOREIGN_KEY_DELETE_ACTION) // FIXME
+                .assertionModifier(SiardArchiveAssertions.IGNORE_FOREIGN_KEY_UPDATE_ACTION) // FIXME
```

Comment:
There are a bunch of fixme's sprinkled in the code base without any explanation what actually should be fixed...

we should either fix it now, create a github issue or explain what should be fixed in a short comment